lib/src/patterns/command/MacroCommand.dart

```diff
@@ -57,15 +54,14 @@ class MacroCommand extends Notifier implements ICommand
    *
    * -  Param [note] - the [INotification] object to be passed to each 'SubCommand'.
    */
-  void execute( INotification note )
-  {
-      for ( Function commandFactory in subCommands ) {
-          ICommand commandInstance = commandFactory();
-          commandInstance.initializeNotifier( multitonKey );
-          commandInstance.execute( note );
-      }
+  void execute(INotification note) {
+    for (Function commandFactory in subCommands) {
+      ICommand commandInstance = commandFactory();
+      commandInstance.initializeNotifier(_multitonKey);
```

Comment:
Change `_multitonKey` back to `multitonKey`.

Reply:
Its done. Thanks for hint.